### DISCUSSION TITLE
Macos-latest stuck on Starting your workflow run

### DISCUSSION THREAD

**Discussion by goshiz** [community/community#25368] [Actions]
<p>Hi,</p>
<p>I’ve been around other subject but I can’t find an issue like mine. I setup a workflow to build a C++ program on Linux, Windows &amp; MacOS.</p>
<p>Linux &amp; Windows is fine, but MacOS won’t even start and stay on Starting your workflow run… forever.</p>
<p>I can’t see anything wrong is my workflow:</p>
<p>build-macos:<br>
runs-on: macos-10.15<br>
steps:<br>
- uses: actions/checkout@v1<br>
with:<br>
submodules: recursive</p>
<p>Thanks a lot :slight_smile:</p>

**Comment by iambaim** [comment]
<p>Hi <a class="mention" href="/goshiz">@goshiz</a>,</p>
<p>This happens in all my workflows as well. MacOS runners simply won’t start.</p>
<p>Github status (<a href="https://www.githubstatus.com" rel="nofollow noopener">https://www.githubstatus.com</a>) says nothing wrong though.  I wonder what is really happening…</p>

**Comment by goshiz** [comment]
<p>Hi,</p>
<p>Thanks for you answer. Since when did you have this issue ?<br>
Is this just bad luck that I’m trying this now, or it’s like this for days ?</p>
<p>Thanks</p>

**Comment by iambaim** [comment]
<p>Just happened today, around afternoon-ish (CET time). Was working fine earlier in the morning and yesterday too.</p>

**Comment by alepauly** [comment, marked as answer]
<p><a class="mention" href="/goshiz">@goshiz</a>, <a class="mention" href="/iambaim">@iambaim</a> - we discovered an issue affecting newer orgs and running Mac workflows, we’re working on fixing it right now. It’s very localized so I’m guessing that’s what you’re seeing.</p>

**Comment by goshiz** [comment]
<p>Thanks you  ! I can confirm that this morning (in Europe) it’s working !</p>

**Comment by iambaim** [comment]
<p>Thanks! It’s all good now.</p>
